tests/ui/span/regions-close-over-type-parameter-2.stderr

```diff
@@ -6,7 +6,6 @@ LL |         let tmp0 = 3;
 LL |         let tmp1 = &tmp0;
    |                    ^^^^^ borrowed value does not live long enough
 LL |         repeater3(tmp1)
-   |         --------------- borrow later captured here by trait object
```

Comment:
This suggestion is was never valid. `repeater3` is what's responsible for doing the unsizing, and the fact that we were reporting this as capturing `A` just because it has the signature `fn<'a, A: 'a>(A) -> Box<dyn Tr + 'a>` is kinda... coincidental.